web/src/app/admin/skills/skills.component.html

```diff
@@ -1 +1,65 @@
-<p>skills</p>
+<div class="container">
+    <div class="panel panel-primary">
+        <div id="isCreate" class="panel-heading">
+            {{ 'SKILLS.CREATE_SKILLS' | translate }}
+        </div>
+        <div *ngIf="isUpdate" id="updatePanel" class="panel-heading">
+           {{ 'SKILLS.UPDATE_SKILLS' | translate }}
+        </div>
+        <div class="panel-body">
+            <form (ngSubmit)="onSubmit(skillsForm)" #skillsForm="ngForm" class="form-horizontal">
+                <label class="col-sm-2" for="name"> {{ 'SKILLS.NAME' | translate }}</label>
+                <div class="col-col-3">
+                    <input type="hidden" name="skillId" [(ngModel)]="skillsModel.skillId">
+                    <input type="text" class="col col-6" id="skills" id="name" name="name" [(ngModel)]="skillsModel.name" required minlength="3"
+                        maxlength="20" element-validation (blur)="checkNameExitsOnBlur()">
+                <small *ngIf="isUserExists" id="nameExists" class="text-danger">T{{ 'SKILLS.MESSAGE' | translate }}</small>
+                </div><br>
+                <label class="col-sm-2" for="description">{{ 'SKILLS.DESCRIPTION' | translate }} </label>
+                
+                <div class="form-group">
+                    <input type="text" class="col-col-6" id="description" name="description" [(ngModel)]="skillsModel.description">
+                </div>
+                <button type="submit" id="btnSave" class="btn btn-primary" >{{ 'SKILLS.SAVE' | translate }}</button>
+                <button *ngIf="isUpdate" id="btnUpdate" type="submit" class="btn btn-primary">{{ 'SKILLS.UPDATE' | translate }}</button>
+            </form>
+        </div>
+    </div>
+</div>
+<div class="container">
+    <h3>{{ 'SKILLS_DETAILS.SKILLS_DETAILS' | translate }}</h3>
+    <table class="table table-bordered">
+        <thead>
+            <tr>
+                <th width=400> {{ 'SKILLS_DETAILS.SKILLS' | translate }}</th>
+                <th width=400> {{ 'SKILLS_DETAILS.DESCRIPTION' | translate }}</th>
+                <th width=400> {{ 'SKILLS_DETAILS.ACTION' | translate }}</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr *ngFor="let skills of skills">
```

Comment:
change iteration variable name to singular

Reply:
done